ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -67,6 +66,8 @@ public void contactInfoTest() throws Exception {
         patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         patientDashboardPage.clickOnShowContact();
         assertTrue(patientDashboardPage.getTelephoneNumber().contains("111111111"));
+        homePage = patientDashboardPage.goToHomePage();
+
```

Comment:
Do we really need line 70 which was not there before?

Reply:
Resolved thanks